rdd/src/main/scala/eval/RDDResult.scala

```diff
@@ -0,0 +1,29 @@
+package maml.rdd.eval
+
+import maml.ast._
+import maml.error._
+import maml.eval._
+import maml.eval.tile._
+import maml.rdd.ast._
+
+import geotrellis.raster._
+import geotrellis.spark._
+
+import cats.data.{NonEmptyList => NEL, _}
+import Validated._
+
+import org.apache.spark.rdd._
+
+import scala.reflect.ClassTag
+
+
+case class SpatialRDDResult(res: TileLayerRDD[SpatialKey]) extends Result {
+  def as[T](implicit ct: ClassTag[T]): Interpreted[T] = {
+    val cls = ct.runtimeClass
+    if (classOf[RDD[(SpatialKey, Tile) with Metadata[TileLayerMetadata[SpatialKey]]]] isAssignableFrom cls)
```

Comment:
Should be `classOf[RDD[(SpatialKey, Tile)] with Metadata[TileLayerMetadata[SpatialKey]]]`